### PR TITLE
Lesson 2-addition

### PR DESCRIPTION
**Pull request by bramakrishna16**

### CODE REVIEW

File: add.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const addFunc=(a,b)=>{
+return a+b;
```
Comment (allopez7) [inline add.js]: Please change a, b to more descriptive names.

File: addtest.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const add=require('./add.js');
+
+const addTest=(a,b,res)=>{
```
Comment (allopez7) [inline addtest.js]: Change a,b here as well.

File: add.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const addFunc=(num1,num2)=>{
+return num1+num2;
```
Comment (songz) [inline add.js]: indent.

File: addtest.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const add=require('./add.js');
+
+const addTest=(num1,num2,res)=>{
+  if (add(num1,num2)==res){
+    console.log("Test case passed!!");
+  }
+  else{
+  console.log("Failed");
```
Comment (songz) [inline addtest.js]: indent

**Pull request merged**